### PR TITLE
SD-3368 Added a minimize action on authoring screen

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
SD-3368

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1: 
I'm refactoring authoring e2e tests so I will add a test for this button, once the PR will be merged.

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html [hunk 1]
```diff
@@ -47,14 +47,18 @@
 			 ng-if="highlight"
 			 translate>Export</button>
 
-        <div ng-if="item._type !== 'legal_archive' && itemActions.multi_edit" class="navbtn dropdown strict multiedit-dropdown pull-left" title="{{ :: 'Multiedit' | translate }}" dropdown>
+        <button class="navbtn navbtn-hover strict" title="{{ :: 'Minimize' | translate }}" ng-click="minimize()">
+            <i class="big-icon-minimize"></i>
+        </button>
+
+        <div ng-if="item._type !== 'legal_archive' && itemActions.multi_edit" class="navbtn dropdown strict multiedit-dropdown pull-left strict" title="{{ :: 'Multiedit' | translate }}" dropdown>
```
Comment (ioanpocol) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: Too long line

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html [hunk 2]
```diff
@@ -47,14 +47,18 @@
...
             <button class="dropdown-toggle" dropdown-toggle>
                 <i class="big-icon-multiedit"></i>
             </button>
             <div class="dropdown-menu pull-right" sd-multiedit-dropdown></div>
         </div>
 
-        <div id="authoring-extra-dropdown" ng-if="item._type !== 'legal_archive'" class="navbtn dropdown pull-left" dropdown>
+        <div id="authoring-extra-dropdown" ng-if="item._type !== 'legal_archive'" class="navbtn dropdown pull-left strict"  title="{{ :: 'More actions' | translate }}" dropdown>
```
Comment (ioanpocol) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: Too long line

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**